resources/build/README.md

```diff
@@ -12,9 +12,9 @@ including this file...
 
 ## Various tools
 
-### build-utils.sh
+### build-utils.sh, builder.inc.sh
 
-See [build-utils.md](build-utils.md)
+See [builder.md](builder.md)
```

Comment:
Since `builder.md` is now under `docs/` this link doesn't work.

Reply:
Fixed, thanks for spotting that! (And it was me who moved that doc too...)